feature-configs/cn-ran-overlays/hacks/pre_deploy.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -e
+
+absolute_feature_dir="$(pwd)/$(dirname $0)"
+feature=$(basename ${absolute_feature_dir})
+cluster_type="${CLUSTER_TYPE:-standard}"
```

Comment:
For future code maintenance it would be helpful to list at the top the set of environment variables that are used as inputs to this script.